### ISSUE TITLE
Add source website to notifications sent out from iRecord

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/iRecord#441]
By coincidence there have been two or three emails today about the notifications that people receive from iRecord in relation to records that they may have added via other systems (today's examples concerned UKBMS and Sewbrec).

One issue is that the notifications from iRecord do not acknowledge the source of the record. Proposed enhancement:
- Add the website name to the top of the notification for any record arriving via a website that isn't iRecord

The other issue is that the record IDs in the notifications are hyperlinks to the original record on iRecord, but not all records are visible on iRecord (the Sewbrec records being one example of this). If a user click on this hyperlink they may be taken to a page that says:

![image](https://user-images.githubusercontent.com/16039111/48485641-4a186180-e811-11e8-84e2-feac5e6305e1.png)

That is a worrying message for the recorder to see!

Proposed enhancements:
- Is it possible to hyperlink to the record ID on the originating website rather than on iRecord (but not sure if all originating websites allow for records to be viewed, suspect they don't)
- Alternatively, can the record IDs be presented as hyperlinks only if they are viewable on iRecord? For records that aren't viewable, the record ID should not be a live link, and ideally should refer the recorder back to the originating website in some way.

**Comment by DavidRoy** [comment]
Third potential enhancement is a simple record view for **any** occurrence ID, possibly with a link to originating website?

**Comment by kitenetter** [comment]
Yes, @DavidRoy's suggestion sounds more sustainable and easier to implement. I'm not sure we can actually display the record as that would break the agreement with sites such as sewbrec that do not wish to share for reporting, but a page that says something like "This record originates from website X and is not visible within iRecord" would be good.

**Comment by kitenetter** [comment]
For reference, this is the URL pattern for sewbrec occurrences:
- https://www.sewbrecord.org.uk/details/record?occurrence_id={id}

**Comment by DavidHepper** [comment]
In case it helps with working out where unneeded Notification emails are coming from, my continuous flow of empty lists have first Received headers like this: 
"from localhost.localdomain (wlbrcweb2 [192.171.199.208]) by mail.nerc-wallingford.ac.uk (8.14.4/8.13.8) with ESMTP id wAI6rpE3013466 for <david.hepper@local-software.co.uk>; Sun, 18 Nov 2018 06:53:51 GMT"
